spec/filters/json_spec.rb

```diff
@@ -62,8 +62,9 @@
   end
 
   logstash_version = Gem::Version.create(LOGSTASH_CORE_VERSION)
-
-  if (Gem::Requirement.create('>= 7.0').satisfied_by?(logstash_version) ||
+  
+  # Field Reference can handle special characters since 8.3.0
+  if ((Gem::Requirement.create('>= 7.0').satisfied_by?(logstash_version) && Gem::Requirement.create('< 8.3').satisfied_by?(logstash_version)) ||
```

Comment:
Do we have or can we add test cases for 8.3+ LS versions?

Reply:
we have existing test for [tagging](https://github.com/logstash-plugins/logstash-filter-json/blob/fix_tag_on_failure_test/spec/filters/json_spec.rb#L59-L60) json fail for 8.3+